### PR TITLE
DOC Improve SS4 upgrade guidance

### PR DESCRIPTION
**Pull request by maxime-rainville**
I wanted to know if we provided an SS4 upgrade path and I managed to miss this bit of info even if it was the very first thing on the page.

I've made it an header instead and added extra bits of guidance around archived assets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ SilverStripe Versioned Files Module
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/symbiote/silverstripe-versionedfiles/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/symbiote/silverstripe-versionedfiles/?branch=master)
 [![SilverStripe supported module](https://img.shields.io/badge/silverstripe-supported-0071C4.svg)](https://www.silverstripe.org/software/addons/silverstripe-commercially-supported-module-list/)
 
-**IMPORTANT**: File versioning is [built-in](https://docs.silverstripe.org/en/4/developer_guides/files/file_management/#file-versioning) with SilverStripe 4.x. There is currently no migration path from this module. Current file versions are handled by SilverStripe’s built-in upgrade tooling. Both past file metadata versions (in `FileVersion` table) and content versions (in `assets/_versions`) are retained, but they’re no longer accessible via the CMS UI. If required, those versions can be retrieved by a developer.
+# Upgrading to Silverstripe CMS 4
+
+This is a legacy module for Silverstripe CMS 3.
```
Comment (maxime-rainville) [inline README.md]: We don't have any formal definition of what a legacy module is. The point I'm trying to communicate is you probably should be installing this now if it's not already in your project.

Happy to take this bit out if we feel it's going to cause confusion.

File: README.md [hunk 2]
```diff
@@ -5,7 +5,13 @@ SilverStripe Versioned Files Module
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/symbiote/silverstripe-versionedfiles/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/symbiote/silverstripe-versionedfiles/?branch=master)
 [![SilverStripe supported module](https://img.shields.io/badge/silverstripe-supported-0071C4.svg)](https://www.silverstripe.org/software/addons/silverstripe-commercially-supported-module-list/)
 
-**IMPORTANT**: File versioning is [built-in](https://docs.silverstripe.org/en/4/developer_guides/files/file_management/#file-versioning) with SilverStripe 4.x. There is currently no migration path from this module. Current file versions are handled by SilverStripe’s built-in upgrade tooling. Both past file metadata versions (in `FileVersion` table) and content versions (in `assets/_versions`) are retained, but they’re no longer accessible via the CMS UI. If required, those versions can be retrieved by a developer.
+# Upgrading to Silverstripe CMS 4
```
Comment (robbieaverill) [inline README.md]: Super minor, but this should be a h2 (`##`) since the main heading at the top will be a h1 =)

File: README.md [hunk 3]
```diff
@@ -5,7 +5,13 @@ SilverStripe Versioned Files Module
...
+
+This is a legacy module for Silverstripe CMS 3.
+
+File versioning is [built-in](https://docs.silverstripe.org/en/4/developer_guides/files/file_management/#file-versioning) with Silverstripe CMS 4. There is currently no migration path from this module. Current file versions are handled by Silverstripe CMS’s built-in upgrade tooling. Both past file metadata versions (in `FileVersion` table) and content versions (in `assets/_versions`) are retained, but they’re no longer accessible via the CMS UI. If required, those versions can be retrieved by a developer.
+
+Read the [Keeping archived assets](https://docs.silverstripe.org/en/4/developer_guides/files/file_migration/#keeping-archived-assets) documentation article to configure your Silverstripe CMS 4 project to preserved archived files.
```
Comment (maxime-rainville) [inline README.md]: The previous blurb didn't quite make it explicit that you need to tweak you asset config if you actually want to keep the archived files around.

File: README.md (hunk 2 above)
Comment (maxime-rainville) [reply README.md]:
```suggestion
## Upgrading to Silverstripe CMS 4
```

**Pull request merged**